docs/UserGuide.md

```diff
@@ -285,7 +288,8 @@ Finds all applications whose fields match the keywords provided.
 * An application will be listed only if all the keywords match. The keywords are case-insensitive.
 * Applications with partially matching keywords will not be listed.
     * e.g. searching for the keyword "Goo" will not list applications with "Google".
-* Searches for Deadline must be in the format `MMM DD YYYY HHMM` (e.g. Dec 31 2030 1200).
+* Searches for deadline must be in the format `MMM DD YYYY HHMM`.
+    * e.g. Dec 31 2030 1200 is a valid deadline.
```

Comment:
Good example but maybe I'll extract out and define valid deadline somewhere else